docs/react-v9/contributing/dev-workflow.md

```diff
@@ -91,7 +96,7 @@ When your change is ready, [create a pull request](https://github.com/microsoft/
 
 Common checklist for PR's
 
-- Descriptive title: "feat(package-name): Adding 'multiple' prop to Nav"
+- Descriptive title: "feat:package-name Adding 'multiple' prop to Nav"
```

Comment:
dunno where is this coming from , but it's not valid conventional commit message format.

its always `<type>(<package-name>): <description>`, please revert this change